string tcp_link_name # tcp link name
float64[3] position
float64[3] velocity
float64[3] acceleration 
